### ISSUE TITLE
robot.head.MoveTo( ) fails

### ISSUE THREAD

**Issue by garthz** [personalrobotics/herbpy#11]
Given:

the console was started as follows:
```bash
rosrun herbpy console.py --sim
```

the following command:
```python
robot.head.MoveTo( [0,0] )
```

fails as follows:
```bash
/homes/garthz/ln/local/ros-hydro/src/prpy/src/prpy/base/manipulator.py:81: FutureWarning: comparison to `None` will result in an elementwise object comparison in the future.
  return self.GetRobot().GetDOFValues(self.GetIndices())
/homes/garthz/ln/local/ros-hydro/src/herbpy/src/herbpy/herbpantilt.py:107: FutureWarning: comparison to `None` will result in an elementwise object comparison in the future.
  traj.Insert(0, current_dof_values, config_spec)
---------------------------------------------------------------------------
openrave_exception                        Traceback (most recent call last)
/homes/garthz/ln/local/ros-hydro/src/herbpy/scripts/console.py in <module>()
----> 1 robot.head.MoveTo( [0,0] )

/homes/garthz/ln/local/ros-hydro/src/herbpy/src/herbpy/herbpantilt.pyc in MoveTo(self, target_dof_values, execute, **kw_args)
    110         # Optionally exeucute the trajectory.

    111         if execute:
--> 112             return self.GetRobot().ExecuteTrajectory(traj, **kw_args)
    113         else:
    114             return traj

/homes/garthz/ln/local/ros-hydro/src/prpy/src/prpy/base/robot.pyc in ExecuteTrajectory(self, traj, defer, timeout, period, **kw_args)
    206         needs_base = util.HasAffineDOFs(traj.GetConfigurationSpecification())
    207 
--> 208         self.GetController().SetPath(traj)
    209 
    210         active_manipulators = self.GetTrajectoryManipulators(traj)

openrave_exception: openrave (Assert): [/tmp/buildd/openrave-0.9.1-7-20141006-0124-/src/libopenrave-core/generictrajectory.cpp:214] -> virtual void OpenRAVE::GenericTrajectory::Sample(std::vector<double>&, OpenRAVE::dReal, const OpenRAVE::ConfigurationSpecification&) const, expr: _timeoffset>=0
```

Note that other non-zero small-magnitude inputs also fail.
Including the viewer doesn't change the result.

**Comment by jeking04** [comment]
I believe this is because we need to call ExecutePath rather than ExecuteTrajectory. Fixed in the branch bugfix/MoveTo.

**Comment by mkoval** [comment]
Closing. This was fixed in #14.
